Source/player.cpp

```diff
@@ -163,45 +163,41 @@ struct DirectionSettings {
 	PLR_MODE walkMode;
 	void (*walkModeHandler)(int, const DirectionSettings &);
 };
-
-// clang-format off
 /** Specifies the frame of each animation for which an action is triggered, for each player class. */
 const int PlrGFXAnimLens[enum_size<HeroClass>::value][11] = {

```

Comment:
you should pull in the comments I added here so that people know what this stuff is. The language might need to be modified, but at least it's something.

Reply:
what comments?

Reply:
He submitted a PR to my branch with the changes

Reply:
I cleaned it up, you just have to pull it now